tests/test_PolarBasis2D.py

```diff
@@ -34,39 +34,7 @@ def testPolarBasis2DEvaluate_t(self):
 
         pf = self.basis.evaluate_t(x)
         result = np.array(
-            [ 0.38243133-6.66608316e-18j,  0.3249317 +1.47839074e-01j,
```

Comment:
This is a note to myself to figure out what changed here when I check out the code.

Reply:
Cool, so you have the option to pick what is correct for the algo or science here now. Resolving.